### PR TITLE
docs: Mention ENDLESS_KEY_APP_DEVELOPER_EXTRAS in the developer docs

### PR DESCRIPTION
**Pull request by pwithnall**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -129,6 +129,12 @@ them into kolibri-gnome arguments. Starts kolibri-gnome with a
 specific application ID depending on the URI. This is why a launcher
 process is needed instead of handling these URIs in kolibri-gnome.
 
+#### Debugging
+
+When running the flatpak, you can define `ENDLESS_KEY_APP_DEVELOPER_EXTRAS=1`
```
Comment (starnight) [inline README.md:134]: Should we mention the `Devel` for developing version here, as well?

BTW, the crucial is that the right click is disabled in the application XD

Comment (pwithnall) [reply README.md:134]: I have the non-`Devel` flatpak installed, and the right-click menu works and running with `ENDLESS_KEY_APP_DEVELOPER_EXTRAS=1` enables the ‘Inspect Element’ menu item. Is that not what’s supposed to happen?

Comment (dbnicholson) [reply README.md:134]: The name is created by joining `config.PROFILE_ENV_PREFIX` and `"APP_DEVELOPER_EXTRAS"`. The flathub build gets `PROFILE_ENV_PREFIX` set to `ENDLESS_KEY_` while the devel flatpak (the one built in tree) gets `ENDLESS_KEY_DEVEL_`.

So, for the production flatpak it's `ENDLESS_KEY_APP_DEVELOPER_EXTRAS` while it's `ENDLESS_KEY_DEVEL_APP_DEVELOPER_EXTRAS` for the devel flatpak.

Comment (dylanmccall) [reply README.md:134]: I added another PR that'll make it a little easier to reach the web inspector: https://github.com/endlessm/endless-key-flatpak/pull/78. And I just realized I didn't need to write that bit of README over there :)

**Comment by pwithnall** [comment]
Fixed a better way by #78.

**Pull request closed without merging**